### DISCUSSION TITLE
Docs on setting up dev environment on Mac

### DISCUSSION THREAD

**Discussion by shettyh** [Kong/kong#9241] [Help] [answered]
Hi,

Is there any docs on how to setup developer environment natively on Mac ( Apple Silicon ) ? 

Thanks

**Comment by windmgc** [comment, marked as answer]
@shettyh I would suggest you try https://github.com/kong/gojira which runs smoothly on my M1 Mac, it is using docker & docker-compose as a developer environment

**Reply by shettyh** [reply]
Great, thanks. Will try
